--- bin/jetty.sh.dist	2013-06-14 16:23:55.315174000 -0700
+++ bin/jetty.sh	2013-06-14 16:25:05.734680000 -0700
@@ -11,7 +11,7 @@
 
 # Configuration files
 #
-# /etc/default/jetty
+# /etc/default/jetty-solr
 #   If it exists, this is read at the start of script. It may perform any 
 #   sequence of shell commands, like setting relevant environment variables.
 #
@@ -137,7 +137,7 @@
 ##################################################
 # Read any configuration files
 ##################################################
-for CONFIG in /etc/default/jetty{,8} $HOME/.jettyrc; do
+for CONFIG in /etc/default/jetty-solr $HOME/.jettyrc; do
   if [ -f "$CONFIG" ] ; then 
     readConfig "$CONFIG"
   fi
